docs/refarch/boundedContexts/thirdPartyApi/index.md

```diff
@@ -152,15 +152,9 @@ The PISP/DFSP User receives a confirmation request  to unlink accounts from thei
 
-The Third Party API BC sends a notification via the Notifications BC to the DFSP Payee system to check that the transaction request has a valid agreement. If a valid agreement is in place, the DFSP Payee system issues a Transaction ID and returns a transaction update to the Switch via the Third Party API BC. The Third Party API BC updates the Transaction state via the 3rd-Party Initiated Payment BC, and sends a notification to the PISP noting the Transaction state via the Notifications BC.
-
-The DFSP Account Holder initiates a Agreement/Quote process via the FSPIOP BC, updates the Third Party API BC, and issues a challenge to the PISP User requesting them to agree to the terms of the Agreement/Quote by signing the challenge using their device-embedded private key. The PISP system is issued with a notification of the pending agreement via the Notifications BC.  Once the PISP User agrees to the terms of the DFSP Agreement/Quote, the signed agreement is returned to the PISP system which updates the Transaction state via the Third Party API BC and issues an Authorisation Request.
-
-The Third Party API BC issues the Authorisation Request to the DFSP Account Holder via the Notifications BC requesting verification of the signed Challenge.  The DFSP Account Holder acknowledges receipt of the Notification via the Notifications BC, and runs through the Authentication/Authorization process (lookup transaction verification credential and validate against the private key on the PISP User's device), and thereafter updates the Verification using a PUT instruction to the Third Party API BC which updates the Transaction in concert with the 3rd-Party Initiated Payments BC.
-
-Once the Verification Request has been fulfilled, the Third Party API BC sends an Authorisation Accepted notification to the DFSP Account Holder via the Notifications BC.  The DFSP Account Holder initiates a transfer per the Transation instructions, via the FSPIOP BC Transfer flow.  The FSPIOP BC fulfils and ends the transfer process, and issues a final PUT status update to the DFSP Account Holder. On receipt of the final update, the DFSP Account Holder sends a PATCH update to the Third Party API BC which in turn triggers it to update the Transaction in concert with the 3rd-Party Initiated Payments BC.  Finally, the PISP User is issued with a Transaction Completed notification via the Notification BC.
+Typical applications would include online and instore environments where a third-party platform such as PayPal is used to execute payments.
```

Comment:
I would maybe phrase this slightly differently:

Some applications of Third Party Payment Initiation include:
- Peer to Peer Payments (e.g. GPay in India)
- Online checkouts for seamless end-user ux (e.g. PayPal)
- Payroll Clearing Software

Reply:
Hi Lewis:  Perfect, I will make the changes as requested.  Thanks.  (P.S.: work has been WIP, just not pushed yet... )